docs/topics/javascript.md

```diff
@@ -197,7 +196,24 @@ Properties of `Job`
 - `storage: JobStorage`: Preferences for the way this job attaches to the build storage
 - `docker: JobDockerMount`: Preferences for mounting a Docker socket
 - `serviceAccount: string`: The name of the service account to use (if you need to override the default).
-- `annotations: {[key: string]:string}`: Name/value pairs of annotations to add to the job's pod
+- `resourceRequests: JobResourceRequest`: CPU and memory request resources for the job pod container.
+- `annotations: JobResourceLimit`: CPU and memory limit resources for the job pod container.
```

Comment:
The key here should be `resourceLimits`, correct?

Also, it looks like the `annotations: {[key: string]:string}: Name/value pairs of annotations to add to the job's pod` line may have inadvertently been removed?

Reply:
Yes! My bad, when adding the changes with git. Sorry

Reply:
No worries!  Thanks for the feature and documentation!